build_resources/linux/apparmor-profile

```diff
@@ -0,0 +1,9 @@
+abi <abi/4.0>,
+include <tunables/global>
+
+profile hathor-wallet /opt/Hathor\ Wallet/hathor-wallet flags=(unconfined) {
```

Comment:
Why are you using `unconfined` flag? Did you try other flags?

As far as I understood, this flag disables apparmor protection when using our application, if I'm not wrong. If there are other flags that don't disable completely but limits the application in a way that the wallet works fine, it would be better.

Did you see any other possibility?

Reply:
I didn't apply any other flag for test. I don't know any other possibility, to be honest. This whole file is a template provided by the Ubuntu team to work around the apparmor limitation. In general lines this profile is only identifying the app as "hathor-wallet" and because it is not an anonymous app it can make use of `userns`.

Reply:
Changed to `default_allow`, it is the must correct flag value. For more details refer to: https://manpages.ubuntu.com/manpages/noble/man5/apparmor.d.5.html